File: java/domain/src/main/java/com/youknow/domain/model/Movie.java
package com.youknow.domain.model;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

@AllArgsConstructor
@Data
public class Movie {
    private boolean adult;
    private List<String> genres;
    private String homepage;
    private int id;
    private String overview;
    private float popularity;
    private String posterPath;
    private String releaseDate;
    private int runtime;
    private String status;
    private String tagline;
    private String title;
    private float voteAverage;
    private int voteCount;
}

File: kotlin-rx/domain/src/main/java/com/youknow/domain/model/Movie.kt
package com.youknow.domain.model

data class Movie (
    val adult: Boolean = false,
    val genres: List<String> = listOf(),
    val homepage: String = "",
    val id: Int = 0,
    val overview: String = "",
    val popularity: Float = 0f,
    val posterPath: String = "",
    val releaseDate: String = "",
    val runtime: Int = 0,
    val status: String = "",
    val tagline: String = "",
    val title: String = "",
    val voteAverage: Float = 0f,
    val voteCount: Int = 0
)
